src/gc/gc.cpp

```diff
@@ -36282,7 +36114,7 @@ CFinalize::GcScanRoots (promote_func* fn, int hn, ScanContext *pSC)
 }
 
 #ifdef GC_PROFILING
-void CFinalize::WalkFReachableObjects (gc_heap* hp)
+void CFinalize::WalkFReachableObjects (fq_walk_fn fn)
 {
     BEGIN_PIN_PROFILER(CORProfilerPresent());
```

Comment:
ahh, yes, I missed deleting the one in CFinalize::WalkFReachableObjects. thanks!